### PR TITLE
chores in cleanup tests for liquidation 

### PR DESCRIPTION
**Pull request by rabi-siddique**
The PR does the following:
- Adds waits when verifying auction values to ensure accuracy.
- Segregates tests related to claiming collateral into three separate tests for each vault. This approach has proven to be more reliable, as looping introduces an element of flakiness due to the rapid processing.

### CODE REVIEW

File: test/e2e/specs/liquidation.spec.js [hunk 1]
```diff
@@ -428,28 +428,28 @@ describe('Wallet App Test Cases', () => {
     it('should verify the value of startPrice from the CLI successfully', () => {
       const propertyName = 'book0.startPrice';
       const expectedValue = '9.99 IST/ATOM';
-
+      cy.wait(3000); // eslint-disable-line cypress/no-unnecessary-waiting
       cy.verifyAuctionData(propertyName, expectedValue);
```
Comment (frazarshad) [inline test/e2e/specs/liquidation.spec.js:432]: cant we just add this line inside cy.verifyAuctionData as well?

Comment (rabi-siddique) [reply test/e2e/specs/liquidation.spec.js:432]: We can do that. But it makes `verifyAuctionData` fail for some reason. I did not dig much into but perhaps I will then to `cy.wait().then()` in verifyAuctionData. Perhaps

Comment (rabi-siddique) [reply test/e2e/specs/liquidation.spec.js:432]: Yes, it is required there

Comment (frazarshad) [reply test/e2e/specs/liquidation.spec.js:432]: please update it as well

**Review by frazarshad — APPROVED** [review]

**Pull request merged**